packages/helper-image/src/image-helper.browser.js

```diff
@@ -0,0 +1,145 @@
+/**!
+*
+* Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+*/
+
+/* global Uint8Array, FileReader */
+
+const ExifImage = require(`exif`).ExifImage;
+
+/**
+* Draws the image on the canvas so that the thumbnail
+* could be generated
+* @param {Object} options
+* @returns {Object}
+*/
+export function drawImage(options) {
+  // save current context before applying transformations
+  options.ctx.save();
+  let rad;
+  // convert degrees to radians
+  if (options.flip) {
+    rad = options.deg * Math.PI / 180;
+  }
+  else {
+    rad = 2 * Math.PI - options.deg * Math.PI / 180;
+  }
+  // set the origin to the center of the image
+  options.ctx.translate(options.x + options.width / 2, options.y + options.height / 2);
+  // rotate the canvas around the origin
+  options.ctx.rotate(rad);
+  if (options.flip) {
+    // flip the canvas
+    options.ctx.scale(-1, 1);
+  }
+  // draw the image
+  options.ctx.drawImage(options.img, -options.width / 2, -options.height / 2, options.width, options.height);
+  // restore the canvas
+  options.ctx.restore();
+}
+
+/**
+* fetches and updates the image file with exif information, required to correctly rotate the image activity
+* @param {Object} file
+* @returns {Promise<Object>}
+*/
+export function fixImageOrientation(file) {
```

Comment:
This doesn't appear to change the image, so `fix` is probably not the right name